doc/api/http2.md

```diff
@@ -694,8 +694,8 @@ added: v8.4.0
   * `weight` {number} Specifies the relative dependency of a stream in relation
     to other streams with the same `parent`. The value is a number between `1`
     and `256` (inclusive).
-  * `getTrailers` {Function} Callback function invoked to collect trailer
-    headers.
+  * `hasTrailers` {boolean} When `true`, the `Http2Stream` will emit the
```

Comment:
Do we update YAML metadata when we add an options field? If yes, there are 4 such additions in this PR.

Reply:
I would rather call this `waitForTrailers`.